Dockerfile.arm64

```diff
@@ -11,42 +11,212 @@
+    libpcap libmnl libnfnetlink libnetfilter_conntrack \
+    ipset \
+    iputils \
+    # Need arp
+    net-tools \
+    # Need kmod to ensure ip6tables-save works correctly
+    kmod \
+    # Also needed (provides utilities for browsing procfs like ps)
+    procps-ng \
+    iproute \
+    iproute-tc \
+    # Needed for conntrack
+    libnetfilter_cthelper libnetfilter_cttimeout libnetfilter_queue \
+    conntrack-tools \
+    # Needed for runit startup script
+    which \
+    # Needed for the cleanup script
+    findutils && \
+    microdnf clean all && \
+    # Install iptables via rpms. The libs must be force installed because the iptables source RPM has the release
+    # version '9.el8_0.1' while the existing iptables-libs (pulled in by the iputils package) has version '9.el8.1'.
+    rpm --force -i /tmp/rpms/iptables-libs-${IPTABLES_VER}.el8.${ARCH}.rpm && \
+    # Install compatible libnftnl version with selected iptables version
+    rpm --force -i /tmp/rpms/libnftnl-${LIBNFTNL_VER}.el8.${ARCH}.rpm && \
+    rpm -i /tmp/rpms/iptables-${IPTABLES_VER}.el8.${ARCH}.rpm && \
+    # Set alternatives
+    alternatives --install /usr/sbin/iptables iptables /usr/sbin/iptables-legacy 1 && \
+    alternatives --install /usr/sbin/ip6tables ip6tables /usr/sbin/ip6tables-legacy 1
+
+# Add mitigation for https://access.redhat.com/security/cve/CVE-2019-15718
+# This can be removed once we update to ubi:8.1
+RUN systemctl disable systemd-resolved
 
 # Copy our bird binaries in
 COPY --from=bird /bird* /bin/
 
-# Copy in the filesystem - this contains felix, calico-bgp-daemon etc...
+# Copy in the filesystem - this contains felix, calico-bgp-daemon, licenses, etc...
 COPY filesystem/ /
 
-# Copy in the calico-node binary
-COPY dist/bin/calico-node-${ARCH} /bin/calico-node
+# On UBI, /sbin/ is a symlink so the above copy clobbers it; restore /sbin as symlinks (some binaries and
+# scripts hard-code /sbin/xyz, for example.
+RUN ln -s /usr/sbin/* /sbin/
 
 COPY --from=bpftool /bpftool /bin
 
-RUN rm /usr/bin/qemu-aarch64-static
+# Copy in the calico-node binary
+COPY dist/bin/calico-node-arm64 /bin/calico-node
+
+# Clean out as many files as we can from the filesystem.  We no longer need dnf or the platform python install
+# or any of its dependencies.
+ADD clean-up-filesystem.sh /
+RUN /clean-up-filesystem.sh
+
+# Copy everything into a fresh scratch image so that naive CVE scanners don't pick up binaries and libraries
+# that have been removed in our later layers.
+FROM scratch
+COPY --from=ubi / /
+
+# Add in top-level license file
+COPY LICENSE /licenses
+
+RUN rm /usr/bin/qemu-*-static
 
 CMD ["start_runit"]
 
+# Required labels for certification
+LABEL name="Calico node" \
+      vendor="Project Calico" \
+      version=$GIT_VERSION \
+      release="1" \
+      summary="Calico node handles networking and policy for Calico" \
+      description="Calico node handles networking and policy for Calico" \
+      maintainer="laurence@tigera.io"
```

Comment:
I think you probably want to put yourself as the maintainer on this one, @frozenprocess  :)

Reply:
I was hoping no one would notice this :D